headertest/testing.go

```diff
@@ -222,13 +294,14 @@ func RandBlockID(t *testing.T) types.BlockID {
 }
 
 // FraudMaker creates a custom ConstructFn that breaks the block at the given height.
-func FraudMaker(t *testing.T, faultHeight int64) header.ConstructFn {
+func FraudMaker(t *testing.T, faultHeight int64, bServ blockservice.BlockService) header.ConstructFn {
 	log.Warn("Corrupting block...", "height", faultHeight)
```

Comment:
should this warn not actually be inside the returned function? otherwise it will only ever be called once no? (when providing it)